src/mixins/embed.js

```diff
@@ -105,13 +108,43 @@ export default {
     });
     const metadata = data && data.data.records[0].data.contentMetadata;
     const stakeholders = data && data.data.records[0].data.stakeholders;
+
+    const stakeholdersFirstId = (stakeholders[0] && stakeholders[0].entity && stakeholders[0].entity['@id']) || '';
+    let inputAddress = stakeholdersFirstId;
+    const addressLengthWithoutPrefixAnd1 = 38;
+    if (stakeholdersFirstId.startsWith('did:like:')) {
+      inputAddress = `like1${stakeholdersFirstId.slice(stakeholdersFirstId.length - addressLengthWithoutPrefixAnd1)}`;
+    } else if (inputAddress.startsWith('did:cosmos:')) {
+      inputAddress = `cosmos1${stakeholdersFirstId.slice(stakeholdersFirstId.length - addressLengthWithoutPrefixAnd1)}`;
+    }
+    let stakeholdersValidlikeWallet;
+    if (isValidAddress(inputAddress)) {
+      stakeholdersValidlikeWallet = changeAddressPrefix(inputAddress, 'like');
+    }
+    let address = '';
+    let stakeholdersName;
+    if (stakeholdersValidlikeWallet
+      || (stakeholders && stakeholders[0] && stakeholders[0].entity.name)) {
```

Comment:
we want `stakeholdersValidlikeWallet` to be trueful so it cannot be `||`, we only consider user with wallets